test/integration/show_name_helper_test.rb

```diff
@@ -0,0 +1,16 @@
+require "test_helper"
+
+# Tests of helper module
+class ShowNameHelperTest < IntegrationTestCase
```

Comment:
What I really wanted to do was write functional tests for ShowNameHelper#approved_syn_links and ShowNameHelper#obss_by_syn_links. I couldn't figure out how.
This is a weak substitute, but better than nothing. 
So I will probably merge and push it out. I can always improve it later